Windows/install_scripts/s2e/startup.bat

```diff
@@ -0,0 +1,44 @@
+timeout 5

```

Comment:
Can we call this `launch.bat`? We have (what I think is) an equivalent script for Linux images at https://github.com/S2E/guest-images/blob/master/Linux/s2e_home/launch.sh

We should aim to keep these consistent across OSs - name, git repo, etc.